src/mruudecode.h

```diff
@@ -30,6 +30,37 @@ extern "C" {
 char* mruudecode_do(const char* text, char** ret_binary, size_t* ret_binary_bytes, char** ret_filename);
 
 
+/**
+ *  From here new code from: Christian Schneider <schneider17@gmx.de>  
+ */
+
+
+/* return hex representation of a string*/
+char* mr_print_hex(char* s);
+
+/* delivers one line from a string */
+int   mr_getline (char** line, char* source, char** nextchar);
```

Comment:
I thought about this too :-)
I'll move all function declarations which are only helpers or internally needed to the .c file so that at the end only the really public needed ones will be in .h file. Maybe then only the .c file needs to be changed from master? I'll see.